src/app/admin/booth/add-booth/add-booth.component.ts

```diff
@@ -26,21 +26,20 @@ export class AddBoothComponent implements OnInit {
   addBooth() {
     let boothForm:any = this.boothForm.value;
 
-    for (let i = 0; i < boothForm.amount; i++)
-    {
+    if(boothForm.amount > this.maxAmountOfBoothsAllowed) {  }
```

Comment:
Missing:
error => {
              this.alerts.push({
                class: "text-center",
                type: "danger",
                msgStrong: "Fejl!",
                msg: "Et maximum af " + maxAmountOfBoothsAllowed + " stande kan blive lavet ad gangen.",
                timeout: 5000
              });